modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/notification/term/contributor/SalesAgentNotificationTermProvider.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.commerce.internal.notification.term.contributor;
```

Comment:
@gabrielwas should we change this package to be `term.provider`?

Reply:
Thx!